pages/index.js

```diff
@@ -210,7 +220,7 @@ export default function Home({ total, today }) {
     <>
       <PageHead />
 
-      <div className="bg-primary-low dark:drop-shadow-none dark:bg-primary-high mb-8 p-8 drop-shadow-md">
+      <div className="p-8 mb-8 bg-primary-low dark:drop-shadow-none dark:bg-primary-high drop-shadow-md">
```

Comment:
These changes look the same but some classes have been moved around? This makes it difficult to review, please undo changes like this

Reply:
I have undo the changes back to how it was originally. I hope it the review easier